### DISCUSSION TITLE
UDP Stream to Liquidsoap examples?

### DISCUSSION THREAD

**Discussion by jschmall** [rtl-airband/RTLSDR-Airband#354] [Q&A]
Since I already use Liquidsoap for my OP25 instance, I would love to also use LS to ingest UDP streams from RTLSDR-Airband. I know that I can output a UDP stream but I'm not sure how to bring that into LS. Any examples or suggestions on where to begin?

**Comment by charlie-foxtrot** [comment]
I haven't used Liquidsoap so can't give you any examples.

I use `netcat` to convert the UDP stream into `stdin`, maybe that can help you?  Here is an example:

`nc -l -u -p ${PORT} | <downstream_process>`

**Reply by jschmall** [reply]
I noticed that in the documentation. I might look into that implementation.

In the mean time, I’ve decided to push all streams to a pulseaudio server, then use liquidsoap to ingest those streams and process them. I’m trying to figure out how to separate those pulse streams within liquidsoap but that’s a problem for another day.
